react/src/components/Widget/WidgetContainer.tsx

```diff
@@ -218,17 +219,17 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
         );
         setCryptoAmount(convertedObj.string);
       } else if (!isFiat(currency)) {
-        setCryptoAmount(amount?.toString());
+        setCryptoAmount(cryptoAmount?.toString());
       }
-    }, [price, currencyObj, amount, currency, randomSatoshis, to]);
+    }, [price, currencyObj, cryptoAmount, currency, randomSatoshis, to]);
 
     return (
       <React.Fragment>
         <Widget
           to={to}
           {...widgetProps}
-          amount={amount}
-          setAmount={setAmount}
+          amount={cryptoAmount}
+          setAmount={setCryptoAmount}
```

Comment:
What did renaming these do for us?

Reply:
we already had these values in the component here: 
`const [cryptoAmount, setCryptoAmount] = useState<string>();` 

I think they mean the same thing as amount and setAmount, just were named differently to avoid errors with props with the same name ? 

because of the useEffects not correctly set the component wasn't passing the values correctly so I did this fix to address correctly the value of the amount